configuration/pih/liquibase/sql/global_metadata.sql

```diff
@@ -0,0 +1,96 @@
+#
+DROP PROCEDURE IF EXISTS initialize_global_metadata;
+#
+CREATE PROCEDURE initialize_global_metadata()
```

Comment:
New procedure to initialize a bunch of global metadata that can be used in reports